docs/src/main/asciidoc/_admin-guide/Galleon_provisioning.adoc

```diff
@@ -127,6 +127,7 @@ Basic layers:
 * _cdi_: Support for CDI. Optionally depends on _bean-validation_.
 * _datasources_: Support for datasources.
 * _ee-security_: Support for EE Security. Depends on _cdi_.
+* _jsf_: Support for Jakarta Server Faces. Depends on _web-server_ (defined in the WildFly servlet feature-pack) and optionally on _cdi_ and _bean-validation_
```

Comment:
@yersan as recent user of these new galleon layers I found it extremely hard to understand how to provision a trimmed server after I cloned the WildFly repository; If this is not on purpose, I suggest adding a small paragraph like the following (not necessarily here and probably with better wording):

_After cloning the WildFly repository locally, you can provision a trimmed server leveraging the galleon cli:
`galleon.sh install wildfly:current/snapshot --layers=jsf,datasources-web-server --dir=my-wildfly-server`_

Reply:
Hello @tommaso-borgato I opened this task to take a look at some possible improvements about layers documentation https://issues.redhat.com/browse/WFLY-13744, please add there any comments you think we can add into the documentation, thanks

In the test plan, we discussed how to do it, but I got the impression it is a bit confusing. So, let me add here this note, you don't need to clone the WildFly repo to use the Galleon tools. If you want to use this "in progress" layer, then you need to get this topic branch, but after it is merged you don't need to clone WildFly in you local repository. You could just use `galleon.sh install wildfly:current`.

Reply:
@yersan tnks, I added a comment there ....